news/_posts/2021-05-05-community-survey-results.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Results of community survey – April 2021"
+subtitle: "Thank you for your feedback about the open source project and the contributions"
+date: 2021-05-05 12:00:00
+authors: [MarionFrancois]
+icon: icon-world
+image: /assets/images/2021/05/build-survey-banner.png
+---
+
+The core product team carried out a survey in April, concerning the open source project and for the community. The goal of this survey was to better know and understand the pains and the wishes of the community about the open source project.
```

Comment:
"and for" 
the open source for the community ?

Reply:
Fixed, thank you